Modules/_bisectmodule.c

```diff
@@ -234,21 +234,21 @@ having to sort the list after each insertion. For long lists of items with\n\
 expensive comparison operations, this can be an improvement over the more\n\
 common approach.\n");
 
+static struct PyModuleDef_Slot _bisectmodule_slots[] = {
+    {0, NULL}
+};
 
 static struct PyModuleDef _bisectmodule = {
     PyModuleDef_HEAD_INIT,
-    "_bisect",
-    module_doc,
-    -1,
-    bisect_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
+    .m_name = "_bisect",
+    .m_doc = module_doc,
+    .m_methods = bisect_methods,
+    .m_slots = _bisectmodule_slots,
```

Comment:
If there is no slot, you can leave m_slots as NULL.

Reply:
Thanks!